### ISSUE TITLE
Suspicious messages from script utils/translations/update_translations.sh

### ISSUE THREAD

**Issue by ordtrogen** [monero-project/monero#3637]
Do these message indicate things need to be fixed in the source files? (just including a screen shot of part of the output to solicitate feedback)

![image](https://user-images.githubusercontent.com/15184875/38769988-1eb189fc-400c-11e8-826f-bdff4bff3dfc.png)

**Comment by erciccione** [comment]
Correct, but that's beyond my capabilities. Somebody familiar with both QT and C++ should give it a look

**Comment by selsta** [comment]
Obsolete by https://github.com/monero-project/monero/pull/10675.
